src/GraphQL/Introspection/ISchemaFilter.cs

```diff
@@ -3,40 +3,52 @@
 
 namespace GraphQL.Introspection
 {
+    /// <summary>
+    /// Represents an filter for information returned from introspection queries.
+    /// Can be used to hide information, such as graph types, from clients that have an inadequate permission level.
+    /// </summary>
     public interface ISchemaFilter
```

Comment:
We already have comments for this file in `develop`.

Reply:
I've backported/merged the comments from `develop` back here.  Please re-review.